src/RegistryCoordinator.sol

```diff
@@ -471,6 +471,8 @@ contract RegistryCoordinator is
         // If the operator wasn't registered for any quorums, update their status
         // and register them with this AVS in EigenLayer core (DelegationManager)
         if (_operatorInfo[operator].status != OperatorStatus.REGISTERED) {
+            require(lastDeregistrationTimestamp[operator] + REREGISTRATION_DELAY < block.timestamp, "RegistryCoordinator._registerOperator: operator cannot reregister yet");
```

Comment:
Can we have an `updateBLSPublickey` that is outside of the registration flow?  we should avoid the 7 day re-registration delay if possible